### PR TITLE
MRG: switch HF-SEF to v1.1 on zenodo

### PR DESCRIPTION
**Pull request by jjnurminen**
#### What does this implement/fix?
Update HF-SEF dataset to a new version (with updated trans) located on zenodo.org.

### CODE REVIEW

File: mne/datasets/hf_sef/hf_sef.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def data_path(dataset='evoked', path=None, force_update=False,
     destdir = op.join(path, 'HF_SEF')
 
     urls = {'evoked':
-            'https://osf.io/25f8d/download?version=1',
+            'https://zenodo.org/record/3523071/files/hf_sef_evoked.tar.gz',
```
Comment (agramfort) [inline mne/datasets/hf_sef/hf_sef.py]: I am surprised you don't have to update any hash

Comment (jjnurminen) [reply mne/datasets/hf_sef/hf_sef.py]: I am not surprised that I forgot to update the hash...

**Comment by jjnurminen** [comment]
I also included the hash for the raw data archive, which was missing (not sure why).

**Comment by larsoner** [comment]
Both datasets download, verify, and extract just fine locally, thanks @jjnurminen

**Comment by agramfort** [comment]
thanks Jussi !

>

**Pull request merged**